server/views/pages/appointments/appointment/movement-slip.njk

```diff
@@ -20,7 +20,12 @@
 
             <div class="govuk-button-group govuk-!-display-none-print">
                 {{ govukButton({
-                    text: "Print movement slip" + ("s" if appointment.attendees.length > 1 ),
+                    text: "Print one slip per page",
+                    classes: "govuk-button--blue separate-slips-print no-js-hidden",
+                    preventDoubleClick: true
+                }) }}
+                {{ govukButton({
```

Comment:
Does it need to hide this button if only one attendee?

Reply:
I did think about that but there was nothing on the ticket about it - when it's one attendee the buttons basically just do the same thing. Can ask in standup